pkg/api/controllers/volume_test.go

```diff
@@ -73,6 +73,9 @@ func NewFakeVolumePortal() *VolumePortal {
 	mockClient.On("DeleteVolume", ctx.Background(), &pb.DeleteVolumeOpts{
 		Context: c.NewAdminContext().ToJson(),
 	}).Return(&pb.GenericResponse{}, nil)
+	mockClient.On("GetMetrics", ctx.Background(), &pb.GetMetricsOpts{
```

Comment:
Please check whether to have metrics mock here.

Reply:
not needed, removed